blog/2020-10-06-monero-bitcoin.md

```diff
@@ -0,0 +1,190 @@
+They construct this transaction in such a way that it may only be included in the blockchain at time `> t_1`.
+This transaction can be used by either party as a mechanism to make the redeem impossible and lock the protocol in a state where only the refund and punish paths can be activated.
+- Alice constructs and shares with Both an adaptor signature on a refund transaction spending from the output of the cancel transaction to `address_b`.
+She constructs it in such a way that if Bob ever decides to refund, he will need to decrypt it using his monero spend key `s_b`, leaking it to her when publishing the transaction to the refund blockchain.
+This transaction does not need to be timelocked, because it depends on the publication of the cancel transaction, which already depends on time being `> t_1`.
+- Bob signs a punish transaction spending from the cancel transaction output to `address_a`, but which can only be published at time `t_2`.
+He shares this with Alice, to allow her to punish him by taking the Bitcoin in exchange for nothing, in the case that Bob remains inactive for too long.
+This transaction exists to prevent a situation where both Alice and Bob have locked up their coins, but Bob never shares the necessary adaptor signature for Alice to redeem the bitcoin, nor does he refund it himself.
+Without this transaction, Alice could be unfairly punished for Bob's inactivity, with the money on both chains being locked up indefinitely.
+With this transaction, Alice can deal with Bob's possible reckless behaviour and Bob is incentivised to act at least before time `t_2`.
+
+#### Going on-chain
+
+With Alice's cancel transaction signature and refund transaction encrypted signature, Bob can now safely lock up his bitcoin using the lock transaction he constructed before.
+This is because he will be able to refund if Alice stops cooperating.
+
+To do so, he asks his Bitcoin wallet to sign the lock transaction and broadcasts it to the network.
+Alice will eventually see that the transaction has been included in a block, and may wait some extra time to see her desired number of confirmation on that transaction.
+
+Alice can now lock up her monero, because she is safe knowing that she will be able to refund if Bob refunds, and that she will be able to just take the bitcoin if Bob never acts again.
+Alice uses her Monero wallet to transfer the agreed upon monero to the output `(S_a + S_b, V_a + V_b)`.
+Bob will be able to see that this has taken place, because he knows `S_a`, `S_b`, `V_a` and `V_b`, and also `v_a` and `v_b`, granting him the ability to see transactions paying to that output.
+If he verifies that the amount of monero is correct and the transaction has enough confirmations, he can proceed with the protocol.
+
+#### One last signature
+
+The current situation allows both parties to abort without any further cooperation, but Alice does not yet have a way to take Bob's bitcoin.
```

Comment:
Except after `t2`, right?

Reply:
Yep, do you think it's worth clarifying?

Reply:
I balked at it, perhaps 

```suggestion
The current situation allows both parties to abort without any further cooperation, but Alice does not yet have a way to take Bob's bitcoin (before the punish timelock).
```
Feel free to not use this though.